docs/channels/utm_tags.rst

```diff
@@ -39,12 +39,12 @@ The following table lists the Google Analytics UTM tags:
      - Description
    * - Campaign Source
      - The referring source of the web activity. It indicates the social network, search engine, newsletter name, or any other specific source driving the traffic.  
-        * Examples: facebook, twitter, blog, newsletters
+        * Examples: ``facebook``, twitter, blog, newsletters
         * UTM code: utm_source
-        * Sample code: utm_source=facebook
+        * Sample code: ``utm_source=facebook``
    * - Campaign Medium
      - UTM tags - Mautic Documentation
-         * Examples: cpc, organic_social
+         * Examples: ``cpc``, organic_social
```

Comment:
As above - why just this one commented and the rest not?

Reply:
it was throwing a vale error but I will comment both instead of having one commented